.github/actions/upload_4C_build/action.yml

```
@@ -13,7 +13,9 @@ runs:
   steps:
     - name: Package build directory
       shell: bash
-      run: tar -cf $GITHUB_WORKSPACE/4C_build.tar $GITHUB_WORKSPACE/build
+      run: |
+        cd ${{ inputs.build-directory }}
+        tar -cf $GITHUB_WORKSPACE/4C_build.tar .
```

Not sure, but I think you need this.
```suggestion
      run: |
        cd $BUILD_DIRECTORY
        tar -cf $GITHUB_WORKSPACE/4C_build.tar .
      env:
        BUILD_DIRECTORY: ${{ inputs.build-directory }}
```

---

I saw that we have this elsewhere. Tbh, I don't understand why this would change things but I believe we had some trouble with this in the past?